prow/ANNOUNCEMENTS.md

```diff
@@ -183,6 +183,16 @@ state and no claims of backwards compatibility are made for any external API.
 Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
+ - *February 16th, 2022* Since prow version `v20220217-362d6ac4a0`, the
+   entrypoint container in a prow job will run as `--copy-mode-only`, instead of
+   `/bin/cp /entrypoint /tools/entrypoint`. Entrypoint images before the mentioned version
+  will not work with `--copy-mode-only`, and entrypoint image since the
+  mentioned version will not work with `/bin/cp /entrypoint /tools/entrypoint`.
+  In another word, prow versions newer than or equal to `v20220217-362d6ac4a0`
+  will stop working with pod utilities versions older than
+  `v20220217-362d6ac4a0`. If your prow instance is bumped by
+  `prow/cmd/generic-autobumper` then you should not be affected.
+   stop working with this old command. In another word, prow 
```

Comment:
```suggestion
   stop working with this old command.
```

Reply:
this version is not exactly true, since as long as prow images are still produced by bazel, this command will still work. Will need to update the image tag mentioned in this breaking change once prow images are produced by ko